main.qml

```diff
@@ -269,7 +269,9 @@ ApplicationWindow {
         middlePanel.paymentClicked.connect(handlePayment);
         middlePanel.sweepUnmixableClicked.connect(handleSweepUnmixable);
         middlePanel.checkPaymentClicked.connect(handleCheckPayment);
-
+        
+        persistentSettings.is_recovering = true;
```

Comment:
I'm afraid this is going to create other bugs. Please provide more info on how to recreate the issues you're having.

Reply:
If you create a new wallet(wallet A) by the walllet-gui, then transform some money from other wallet(wallet B) to A, and you can find this bug,  A can't find any recored. except to recover wallet according to keys or seed words.

Reply:
Is B also created by the GUI?

Reply:
Whether B is created by GUI or by command line, the problem always exists. You can easily test it in your environment. Also is it present in the v0.11.0.0